connector-sdk/core/src/main/java/io/camunda/connector/api/inbound/Health.java

```diff
@@ -23,8 +23,12 @@
 public class Health {
 
   private final Status status;
+  private final Message message;
+
   private final Map<String, Object> details;
 
+  public record Message(Severity severity, String message) {}
```

Comment:
Good point. So would you just go with a single dedicated property `message` and derive the severity from the `status`?

Reply:
Or would you rather have an optional `error` property without any severity? Could be similar to what we had before:
```
public record Error(code, message)
``` 
and we only make us of that in case of an error. No info or any other severity?

Reply:
Yeah, an optional `error` looks reasonable - and since it's a separate class, we will be able to extend it if needed in the future without breaking the API 👍